packages/@sfx/sayt-driver-plugin/README.md

```diff
@@ -0,0 +1,46 @@
+# SF-X Sayt Driver Plugin
+
+This package contains the SF-X Sayt (Search-as-you-type) Driver plugin.
```

Comment:
`SAYT` for English text, but `sayt` or `Sayt` as appropriate for identifier names. Either reword or recapitalize this, and make the same change in the title.